### ISSUE TITLE
Delete pilot 1 references in .Rbuildignore 

### ISSUE THREAD

**Issue by bms63** [RConsortium/submissions-pilot3-adam#180]
This isn't really a package anymore, but just some housecleaning...we should remove these pilot1 references in this file.

![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/91771905-a8b9-47ef-b8c8-563394948600)

**Comment by robertdevine** [comment]
Yes, @bms63, Pilot3 {pilot3utile} has been separated from {pilot1wrappers}. Thx.
